### ISSUE TITLE
Podman compatibility with ANMS

### ISSUE THREAD

**Issue by mfarina1** [NASA-AMMOS/anms#134]
ANMS needs to be able to run perfectly under Redhat Podman, through whatever adaptations are needed, in order for us to request features and be able to eliminate our native Podman adaptations as a source of error and bugs. David Edell has been able to get ANMS on our server, running under podman-compose with only a couple of misbehaving features (the service 'status' depends on docker commands). Networking should all work to allow the ANMS network manager to talk to nodes on our physical network – matter of getting the ION config working with that.

TLDR we need to investigate and hopefully begin supporting Podman instead of Docker CE (NOTE: this is a general ticket used for tracking purposes, it is likely many related tickets will be made stemming from this one)

**Comment by mfarina1** [comment]
According to Brian, one complexity in this is that the current build process uses an intermediate base image to be able to share between different final images; I don’t know how that can be handled in a purely compose-driven build

**Comment by DavidEdell** [comment]
The "dsn-devel" branch now supports podman, and deprecates the original build.sh setup procedure.  See the README in that branch for details.

The compose file now includes build directions to allow building with docker-compose or podman-compose.  A set of placeholder builder containers have been defined to handle building of image dependencies. These contains are started with the system, but will by design exit immediately.

A few limitations include:
- Service status page in the anms-ui has been disabled due to podman incompatibility.  This page may be redesigned to query service status via other means in a future release.
- Rootless podman cannot bind to low-numbered network ports. See the README for instructions on adjusting the default port mappings as necessary via the .env file.
